src/LogikUI/Simulation/Gates/AndGate.cs

```diff
@@ -17,9 +17,9 @@ class AndGate : IComponent
 
         public void GetPorts(Span<Vector2i> ports)
         {
-            ports[0] = new Vector2i(0, 0);
-            ports[1] = new Vector2i(-3, -1);
-            ports[2] = new Vector2i(-3, 1);
+            ports[0] = new Vector2i(-1, 1);
+            ports[1] = new Vector2i(-1, -1);
+            ports[2] = new Vector2i(2, 0);
```

Comment:
This is not correct. The output of the component should be at the mouse position. It's much harder to place components when the mouse is centered around the mouse instead of at the output port